cmd/kube-apiserver/app/server.go

```diff
@@ -208,7 +210,27 @@ func Run(s *options.APIServer) error {
 	if err != nil {
 		glog.Errorf("Failed to create clientset: %v", err)
 	}
-	admissionController := admission.NewFromPlugins(client, admissionControlPluginNames, s.AdmissionControlConfigFile)
+
+	plugins := []admission.Interface{}
+	for _, pluginName := range admissionControlPluginNames {
+		plugin := admission.InitPlugin(pluginName, client, s.AdmissionControlConfigFile)
+		if plugin != nil {
+			plugins = append(plugins, plugin)
+		}
+	}
+
+	informer := informers.CreateSharedNamespaceInformer(client, 5*time.Minute)
```

Comment:
You'll probably want to do something like this to avoid creating more than one informer per type:

```go
informers := map[reflect.Type]framework.SharedIndexInformer{}
informers[reflect.TypeOf(&api.Namespace{})] = namespaceInformer
// more as needed
```